components/map-points.js

```
@@ -20,7 +20,7 @@ export default function MapPoints({ coords }) {
           ? church
           : p.name === "BiGy (Biskupské gymnázium)"
           ? school
-          : p.name === "Nové Adalbertinum"
+          : p.name === "Nové Adalbertinum" || "Biskupská rezidence"
           ? tunel
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)